app/views/components/showGroup.html

```diff
@@ -1,6 +1,6 @@
 <div class="modal-header">
```

Comment:
I think this template should live next to (in the same folder) as the JS component for it. I know the rest of the app isn't structured this way. I'll let you decide how to structure it. The rest of the templates are here so I guess leaving it here is fine. I do love not having to look in a completely different file path to find templates though.